### ISSUE TITLE
ApexLexer logs visible when Apex is the selected language upon starting the designer

### ISSUE THREAD

**Issue by jsotuyod** [pmd/pmd-designer#31]
**Describe the bug**
When the designer is open, **iff** Apex is the currently selected language, the console outputs:

```
Oct 29, 2019 10:42:39 PM apex.jorje.parser.impl.BaseApexLexer dedupe
INFO: Deduped array ApexLexer.DFA22_transition. Found 7272051 shorts which is 13MB not including array overhead. Removed 6615472 shorts which is 12MB not counting array overhead. Took 5 ms.
```

Which seems to ignore the fixes introduced in https://github.com/pmd/pmd/pull/503

**To Reproduce**
1. Launch the designer from run.sh
2. Change the language to Apex
3. Close the desinger
4. Restart the designer from CLI
5. See error

**Expected behavior**
No ApexLexer logs should be visible.

**Comment by adangel** [comment]
True, this log appears. It looks like, the solution in pmd/pmd#503 is not reliable: `Logger.getLogger()` returns a weak reference to the Logger instance, so whatever we configure on this logger instance might not matter, when BaseApexLexer does a own call to `Logger.getLogger()` and is given a different instance. In fact, in the debugger I have seen, that `ApexJorjeLogging.disableLogging()` is called 3 times, and in each of the three times, we got a different instance back for the logger of "ApexLexer" (3 times, because we disable logging via the constructor for ApexParser)

We need to configure java util logging properly via a properties file or via LogManager. I did a short test using the following code:

``` java
        try {
            LogManager.getLogManager().readConfiguration(new ByteArrayInputStream(
                    (BaseApexLexer.class.getSimpleName() + ".level=" + Level.WARNING.getName()).getBytes(StandardCharsets.UTF_8)));
        } catch (SecurityException | IOException e) {
            // TODO Auto-generated catch block
            e.printStackTrace();
        }
```

This code basically configured the log manager with the following properties:
```
apex.jorje.parser.impl.BaseApexLexer.level=WARNING
```

With that, we don't log it anymore. However, this overrides any logging configuration configured elsewhere. Either we need to fix this in PMD or in both PMD and Designer. Maybe both need to properly configure the LogManager in their main classes?
